java/java/src/main/java/org/signal/internal/Native.java

```diff
@@ -86,7 +86,7 @@ private Native() {}
 
   public static native byte[] Aes256GcmSiv_Decrypt(long aesGcmSiv, byte[] ctext, byte[] nonce, byte[] associatedData);
   public static native void Aes256GcmSiv_Destroy(long handle);
-  public static native byte[] Aes256GcmSiv_Encrypt(long aesGcmSiv, byte[] ptext, byte[] nonce, byte[] associatedData);
+  public static native byte[] Aes256GcmSiv_Encrypt(long aesGcmSivObj, byte[] ptext, byte[] nonce, byte[] associatedData);
```

Comment:
Why the Obj naming here?

Reply:
It comes from the Rust source, which renamed a parameter named `aes_gcm_siv` so that it wouldn't shadow the package name.